### ISSUE TITLE
Delete The connection 

### ISSUE THREAD

**Issue by MuhammadAbdullah805** [facebookincubator/facebook-for-magento2#61]
I want to disconnect the Facebook connection  for some reason, But I am unable to delete the connection  from  under **stores->Facebook->setup**  by pressing this button **"Delete Connection"**,
Can you please let me know the steps that I will follow to disconnect/delete the connection  ?

**Comment by benxutech** [comment]
I had trouble with this as well and looking through the code I couldn't see anything that would actually unset the config for installation. It *did* seem to remove the Facebook connection though, but just not on Magento's side.

It may not be the best solution, but what I did was truncate/drop the `facebook_business_extension_config` table and then run `php bin/magento setup:upgrade`. It no longer appeared after that (and I was able to connect to another one again).

**Comment by MuhammadAbdullah805** [comment]
Exactly, I did also change the facebook_business_extension_config table by one of its entry related to enable disable from 1 to 0,It solved my issue . But that is not the ultimate solution. The Extension provider must provide a solution of this problem ASAP

**Comment by tanolalano** [comment]
Same issue here. I'm Using truncate table suggestions @benxutech 
Magento 2.4.3 and Facebook for Magento 1.4.2
